### PR TITLE
Create Favicon closes #88

### LINKED ISSUE #88: Develop logo to use as favicon for client side app
**User Story**
As a user, I want to be able to identify the Feedr app by a logo
As a marketing person, I want to be able to use a logo to brand the feedr app

**Acceptance Criteria**

- Favicon updated on client side app

**Notes**

---

**Why is this feature needed? Please describe the problem your requested feature wants to solve**
Can't use the react logo in prod. (Maybe -> think its a legal issue)

**Describe the solution you'd like**
Develop a logo for the Feedr brand and implement it throughout the app

### PR DESCRIPTION
**Pull request by KimberleyEvans-Parker**
GitHub Issue (If applicable): #88 

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?

Site's favicon is react logo

## What is the new behavior?
Site's icon is Feedr logo

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [ ] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

## Other information
Does not work with Microsoft Edge
Closes #88 

Screenshot:
feedr

Internal Issue (If applicable):
#88

### CODE REVIEW

**Comment by EricPedrido** [comment]
This logo uses several copyrighted logos from other companies. Do we know if it's allowed (legally)?

**Comment by PreetPatel** [comment]
> This logo uses several copyrighted logos from other companies. Do we know if it's allowed (legally)?

Could you update the PR to include screenshots?
Also, reckon since the repo is public, best you'd get is a DMCA takedown request... Happy to go with either but if we're replicating a public open source organisation, would recommend not using them

**Comment by KimberleyEvans-Parker** [comment]
> This logo uses several copyrighted logos from other companies. Do we know if it's allowed (legally)?
> 
> Could you update the PR to include screenshots?
> Also, reckon since the repo is public, best you'd get is a DMCA takedown request... Happy to go with either but if we're replicating a public open source organisation, would recommend not using them

I ended up changing the logo to avoid any copyright issues

File: client/public/index.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
```
Comment (jordansimsmith) [inline client/public/index.html]: I think that we should keep the charset encoding (and maybe the viewport) meta tags.

**Pull request merged**